include/antidote.hrl

```diff
@@ -164,11 +193,24 @@
 -type log_id() :: [partition_id()].
 -type bucket() :: term().
 -type snapshot() :: term().
+-type orddict() :: orddict().
```

Comment:
Replacing orddict() by list() is not solving the issue I think. There are semantic differences.